bundles/org.openhab.binding.shelly/src/main/resources/OH-INF/thing/shellyGen2_relay.xml

```diff
@@ -368,4 +368,18 @@
 		<config-description-ref uri="thing-type:shelly:dimmer-gen2"/>
 	</thing-type>
 
+	<thing-type id="shellyplus10v">
+		<label>Shelly Plus Dimmer 10V</label>
+		<description>@text/thing-type.shelly.shellyplus10v.description</description>
+		<category>DimmableLight</category>
```

Comment:
"DimmableLight" does not exist. Look ,at https://github.com/openhab/openhab-webui/tree/main/bundles/org.openhab.ui.iconset.classic/src/main/resources/icons for a list of all icons.

Documentation: https://www.openhab.org/docs/concepts/categories.html#channel-categories

Reply:
fixed

Reply:
I see you removed the category for thing types. One option would have been to use "Lightbulb" which is a valid category for things.
I let you decide if this is better without category.

Reply:
ya, I thought about this, but a dimmer is not really a Lichtbild even this is one of the main use cases. What do you think?

Reply:
let's go without a mis-leading category